src/main/java/com/treasuredata/client/TDClient.java

```diff
@@ -406,7 +405,8 @@ public boolean existsTable(String databaseName, String tableName)
             return false;
         }
         catch (TDClientHttpException e) {
-            if (e.getStatusCode() == HttpStatus.NOT_FOUND_404) {
+            // NOT_FOUND error
+            if (e.getStatusCode() == 404) {
```

Comment:
`java.net.HttpURLConnection.HTTP_NOT_FOUND` could be used

Reply:
I copied HttpStatus.java, but we can use HttpUrlConnection.xxx instead. Good suggestion.

Reply:
HttpStatus.java contains code name etc., which is useful for debugging and error message. Let me use this.

Reply:
Ah, you copied `HttpStatus.java`.